src/viewer/App.tsx

```diff
@@ -347,6 +371,7 @@ export class AppStateStore {
       id: requestId,
       url: url,
       frames: [],
+      index: new Index({tokenize: "full"}),
```

Comment:
I deemed a `"full"` search to be the best fit.
> index every possible combination
https://github.com/nextapps-de/flexsearch#tokenizer-prefix-search